protocols/direct-fund/direct-fund_test.go

```diff
@@ -76,6 +77,60 @@ func TestNew(t *testing.T) {
 	}
 }
 
+func TestPreFundSideEffects(t *testing.T) {
+	// Construct various variables for use in TestUpdate
+	var s, _ = New(false, testState, testState.Participants[0])
+
+	// Approve the objective, so that the rest of the test cases can run.
+	o := s.Approve()
```

Comment:
You can save some lines by passing in `preapprove = true` on L82

Reply:
[3fd478c](https://github.com/statechannels/go-nitro/pull/188/commits/3fd478cc5a30741d7cb9ea130fafa0d41c3ddf56)